templates/layout/page--ecl-branding--standardised.html.twig

```diff
@@ -0,0 +1,25 @@
+{#
+/**
+ * @file
+ * Theme override to display a single page for 'Standardised' ECL branding.
+ *
+ * There is additional wrapper for 'site_header_container' block and banner with site_name.
+ *
+ * @see ./oe_theme/templates/layout/page.html.twig
+ */
+#}
+{% extends "page.html.twig" %}
+
+{% block site_header_container %}
+  <div class="ecl-site-header-{{ ecl_branding }}__header">
```

Comment:
Can't we just use `standardised` here since we know we are in the Standardised branding?